lib/ibm/helpers/class.rb

```diff
@@ -0,0 +1,19 @@
+# typed: false
+# frozen_string_literal: true
+
+# The base object.
+class Object
+  # Filter out the standard methods and only show the interesting methods.
+  # @return [Array<String>]
+  def im
```

Comment:
This looks handy for dev but I don't think we want to monkeypatch the base `Object` class

Reply:
Yes, it is very handy for dev. It's something I created for a different project that I brought over.  I want this as part of the console experience. It's also very handy for demos and learning. By removing all of the standard methods it easier to discern what the class actually does. We can move it directly into the console file, but then people developing in the cloud plugin won't have it.

Reply:
I'm a heavy :-1: in this in a gem.  Gems should not be patching core classes with their own methods, especially when they are development only.

It might make more sense in a gem like `active_support` or `more_core_extensions` since the purpose of those gems are to patch core classes.

Alternatively, if you are using `pry` you don't need this.  `pry` has the `ls` command which gives you exactly what you are looking for.  See https://gist.github.com/banister/c0fc686ef923c8b87715